tasks/eth/006-MCP-L1/postCheck.s.sol

```diff
@@ -0,0 +1,231 @@
+        OptimismMintableERC20Factory factory = OptimismMintableERC20Factory(prox.OptimismMintableERC20Factory);
+
+        require(factory.BRIDGE() == prox.L1StandardBridge);
+        require(factory.bridge() == prox.L1StandardBridge);
+    }
+
+    /// @notice Asserts that the L1ERC721Bridge is setup correctly
+    function checkL1ERC721Bridge() internal view {
+        console.log("Running chain assertions on the L1ERC721Bridge");
+        L1ERC721Bridge bridge = L1ERC721Bridge(prox.L1ERC721Bridge);
+
+        require(address(bridge.OTHER_BRIDGE()) == Predeploys.L2_ERC721_BRIDGE);
+        require(address(bridge.otherBridge()) == Predeploys.L2_ERC721_BRIDGE);
+
+        require(address(bridge.MESSENGER()) == prox.L1CrossDomainMessenger);
+        require(address(bridge.messenger()) == prox.L1CrossDomainMessenger);
+        require(address(bridge.superchainConfig()) == prox.SuperchainConfig);
+    }
+
+    /// @notice Asserts the OptimismPortal is setup correctly
+    function checkOptimismPortal() internal view {
+        console.log("Running chain assertions on the OptimismPortal");
+
+        OptimismPortal portal = OptimismPortal(payable(prox.OptimismPortal));
+
+        address guardian = cfg.superchainConfigGuardian();
+        if (guardian.code.length == 0) {
+            console.log("Guardian has no code: %s", guardian);
+        }
+
+        require(address(portal.L2_ORACLE()) == prox.L2OutputOracle);
+        require(address(portal.l2Oracle()) == prox.L2OutputOracle);
+        require(address(portal.SYSTEM_CONFIG()) == prox.SystemConfig);
+        require(address(portal.systemConfig()) == prox.SystemConfig);
+        require(portal.GUARDIAN() == guardian);
+        require(portal.guardian() == guardian);
+        require(address(portal.superchainConfig()) == address(prox.SuperchainConfig));
+        require(portal.paused() == SuperchainConfig(prox.SuperchainConfig).paused());
+        require(portal.l2Sender() == Constants.DEFAULT_L2_SENDER);
+    }
+
+    /// @notice Asserts that the ProtocolVersions is setup correctly
+    function checkProtocolVersions() internal view {
+        console.log("Running chain assertions on the ProtocolVersions");
+        ProtocolVersions versions = ProtocolVersions(prox.ProtocolVersions);
+        require(versions.owner() == cfg.finalSystemOwner());
+        require(ProtocolVersion.unwrap(versions.required()) == cfg.requiredProtocolVersion());
+        require(ProtocolVersion.unwrap(versions.recommended()) == cfg.recommendedProtocolVersion());
+    }
+
+    /// @notice Asserts that the SuperchainConfig is setup correctly
+    function checkSuperchainConfig(bool _isPaused) internal view {
+        console.log("Running chain assertions on the SuperchainConfig");
+        SuperchainConfig superchainConfig = SuperchainConfig(prox.SuperchainConfig);
+        require(superchainConfig.guardian() == cfg.superchainConfigGuardian());
+        require(superchainConfig.paused() == _isPaused);
```

Comment:
For this task we never need `isPaused == true`, so we can hardcode the expected value of `false` and remove the input